src-tests/test-rpc.lisp

```diff
@@ -52,11 +61,7 @@
              (let ((server-response (rpcq:rpc-call client "test-method")))
                (is (string= *expected-response* server-response))))
         ;; kill the server thread
-        #+ccl
-        (loop :while (bt:thread-alive-p server-thread)
-              :do (sleep 1) (bt:destroy-thread server-thread))
-        #-ccl
-        (bt:destroy-thread server-thread)))))
+        (kill-thread-slowly server-thread)))))
```

Comment:
Killing them softly.

Reply:
I almost named this function `KILLING-THREAD-SOFTLY` :-)